docs/documentation/community/downloads.md

```diff
@@ -18,10 +18,25 @@ Besides the runtime bundles, Eclipse SmartHome comes with the "Designer", which
 
 ### Releases
 
-The *latest release of Eclipse SmartHome is 0.8*, released on June 15, 2016. For details, please check out the [Release Notes](https://projects.eclipse.org/projects/iot.smarthome/releases/0.8.0/plan). You can download it here:
+The *latest release of Eclipse SmartHome is 0.9*, released on December 6, 2017. For details, please check out the [Release Notes](https://projects.eclipse.org/projects/iot.smarthome/releases/0.9.0/plan). You can download it here:
 
 | Maven repository | [https://repo.eclipse.org/content/repositories/smarthome-releases](https://repo.eclipse.org/content/repositories/smarthome-releases) |
-| p2 repository | [http://download.eclipse.org/smarthome/updates-release/0.8.0/](http://download.eclipse.org/smarthome/updates-release/0.8.0/) (download as [zip](http://eclipse.org/downloads/download.php?file=/smarthome/releases/0.8.0/eclipsesmarthome-incubation-0.8.0-repo.zip)) |
+| p2 repository | [http://download.eclipse.org/smarthome/updates-release/0.9.0/](http://download.eclipse.org/smarthome/updates-release/0.9.0/) (download as [zip](http://eclipse.org/downloads/download.php?file=/smarthome/releases/0.8.0/eclipsesmarthome-incubation-0.9.0-repo.zip)) |
```

Comment:
the zip link points into nirvana 😩

Reply:
Right, because I missed one 0.8.0 occurrence...

Reply:
No, it isn't even published at http://eclipse.org/downloads/download.php?file=/smarthome/releases/0.9.0/eclipsesmarthome-incubation-0.9.0-repo.zip...

Reply:
From looking at https://ci.eclipse.org/smarthome/job/SmartHomeDistributionPublishRelease/6/console I doesn't seem we are publishing the zip at all... Do we?

Reply:
Now we do.